tendrl/gluster_integration/central_store/__init__.py

```diff
@@ -44,3 +44,6 @@ def save_nodecontext(self, node_context):
 
     def save_utilization(self, utilization):
         tendrl_ns.etcd_orm.save(utilization)
+
+    def save_job(self, job):
```

Comment:
@shtripat if the component want to create any job, job is created using job class in common and stored via save method of its own (https://github.com/Tendrl/commons/pull/175)